web_src/js/index.js

```diff
@@ -615,6 +616,21 @@ function initInstall() {
 function initIssueComments() {
   if ($('.repository.view.issue .comments').length === 0) return;
 
+  $('.re-request-review').click(() => {
+    const $this = $('.re-request-review');
+    const $url = $this.data('update-url');
+    const $id = $this.data('id');
+    const $issue_id = $this.data('issue-id');
+    $.post($url, {
+      _csrf: csrf,
+      id: $id,
+      action: '',
+      issue_ids: $issue_id,
+    }, () => {
+      reload();
```

Comment:
Should this prevent default?

Reply:
I'm sorry , But I don't know the meaning of your question, can you  elaborate  it, thanks

Reply:
`event.preventDefault();`

Reply:
thanks for your reminding, I don't know it before.@lafriks